repo/meta/index.json

```diff
@@ -244,6 +244,20 @@
       }
     },
     {
+      "currentVersion":"1.1.0-RC1",
+      "description":"A cluster-wide init and control system for services in cgroups or Docker containers.",
```

Comment:
Our new description is: "A container orchestration platform for Mesos and DCOS" (http://mesosphere.github.io/marathon/)